bokeh/models/tickers.py

```diff
@@ -40,14 +40,16 @@ class ContinuousTicker(Ticker):
 
 class FixedTicker(ContinuousTicker):
     ''' Generate ticks at fixed, explicitly supplied locations.
-
     .. note::
         The ``desired_num_ticks`` property is ignored by this Ticker.
-
```

Comment:
@jfx319 this whitespace is necessary, and is what is currently failing the build (Sphinx is very picky)